### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by rishav3110**
Maps not working

### CODE REVIEW

File: src/components/Footer/Footer.jsx [hunk 1]
```diff
@@ -1,8 +1,86 @@
...
+                            </button>
+                        </div>
+                        <p className={styles.pconnect}>CONNECT WITH US :</p>
+                        <div className={styles.social}>
+                            <div className={styles.imgdiv}><img src="https://res.cloudinary.com/dxafdfvui/image/upload/v1705083563/instapxl_sj0sgp.svg" alt="instagram" /></div>
+                            <div className={styles.imgdiv}><img src="https://res.cloudinary.com/dxafdfvui/image/upload/v1705083563/fbpxl_imhsav.svg" alt="facebook" /></div>
+                            <div className={styles.imgdiv}><img src="https://res.cloudinary.com/dxafdfvui/image/upload/v1705083935/twitterlogo_al9ufe.svg" alt="twitter" /></div>
+                        </div>
+                        <p className={styles.con}>CONTACT US</p>
+                        <p className={styles.gmail}>nitshack@gmail.com</p>
```
Comment (bj-jiwrajka) [inline src/components/Footer/Footer.jsx]: change the email to ```nitshacks.nits@gmail.com```

File: src/components/Footer/Footer.jsx [hunk 2]
```diff
@@ -1,8 +1,86 @@
...
+
+
+                    </div>
+                    <div className={styles.buttondiv}>
+                        <button className={styles.btn2}>
+                            <a href="">VISIT TECNOESIS</a>
+                        </button>
+                    </div>
+                </div>
+                <div className={styles.upper2}>
```
Comment (bj-jiwrajka) [inline src/components/Footer/Footer.jsx]: image

try to make the changes exactly as given in the design

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.module.scss [hunk 3]
```diff
@@ -0,0 +1,274 @@
...
+    padding: 1rem;
+}
+
+.social {
+    display: flex;
+    flex-direction: row;
+    padding-top: 0.81rem;
+    padding-bottom: 1.125rem;
+    align-items: flex-start;
+    gap: 16.194px;
```
Comment (bj-jiwrajka) [inline src/components/Footer/Footer.module.scss]: change px to relative units

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

**Comment by bj-jiwrajka** [comment]
> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> Name 	Status 	Preview 	Updated (UTC)
> **nits-hacks-6-0** 	✅ Ready ([Inspect](https://vercel.com/nitshacks/nits-hacks-6-0/37GgJUaeuMTTMQbCoZKfB5EX3Nom)) 	[Visit Preview](https://nits-hacks-6-0-git-footer-nitshacks.vercel.app) 	Jan 13, 2024 10:48am

image

seems too big, fix it!

**Comment by byt3h3ad** [comment]
(merge conflicts resolve karna)

File: src/components/Footer/Footer.jsx [hunk 4]
```diff
@@ -1,8 +1,110 @@
...
+							</div>
+							<div className={styles.imgdiv}>
+								<img
+									src="https://res.cloudinary.com/dxafdfvui/image/upload/v1705083935/twitterlogo_al9ufe.svg"
+									alt="twitter"
+								/>
+							</div>
+						</div>
+						<p className={styles.con}>CONTACT US</p>
+						<p className={styles.gmail}>nitshacks.nits@gmail.com</p>
```
Comment (bj-jiwrajka) [inline src/components/Footer/Footer.jsx]: make it clickable, use ```mail:to```

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.jsx [hunk 5]
```diff
@@ -1,8 +1,117 @@
...
-    );
-}
-export default Footer;
\ No newline at end of file
+	// const location = {
+	//     address: '1600 Amphitheatre Parkway, Mountain View, california.',
+	//     lat: 37.42216,
+	//     lng: -122.08427,
+	// };
+	return (
```
Comment (bj-jiwrajka) [inline src/components/Footer/Footer.jsx:4]: Format the code and add links to social handles

**Review by bj-jiwrajka — CHANGES_REQUESTED** [review]

File: src/components/Footer/Footer.module.scss (hunk 3 above)
Comment (bj-jiwrajka) [reply src/components/Footer/Footer.module.scss]: fix this too :)

**Pull request merged**